lmctl/project/handlers/brent2dot1/brent_src.py

```diff
@@ -96,6 +96,20 @@ class Sol003LifecycleTree(files.Tree):
     def scripts_path(self):
         return self.resolve_relative_path(Sol003LifecycleTree.SCRIPTS_DIR_NAME)
 
+class Sol005LifecycleTree(files.Tree):
```

Comment:
Sol005 doesn't need to be added to `brent2dot1` - this is a legacy format we no longer use. So you can remove all of the changes you made to files in `brent2dot1`.